test/integration/performance/connection_http_transport_benchmark_test.py

```diff
@@ -0,0 +1,264 @@
+from abc import (
```

Comment:
where would we want benchmarking stored later? put into the integration tests for existing fixture usage.